docs/directives/set-type.md

```diff
@@ -0,0 +1,13 @@
+# Set the data type of a column
+
+The SET-TYPE directive manually set the data type of the column
```

Comment:
Do you mean the type to manually set to must be different from the automatically detected type? For an address data, type detection can give a result as list: [<phone, 10%>, <address, 90%>], but we want to manually set it to [<address, 100%>]. Are you saying if the detected type is already  [<address, 100%>], we don't allow user to set it to the same "type list" again?